cmd/release-controller/sync_verify.go

```diff
@@ -77,8 +77,10 @@ func (c *Controller) ensureVerificationJobs(release *Release, releaseTag *imagev
 			if jobRetries > 0 {
 				jobName = fmt.Sprintf("%s-%d", jobName, jobRetries)
 			}
-
-			job, err := c.ensureProwJobForReleaseTag(release, jobName, verifyType, releaseTag, previousTag, previousReleasePullSpec)
+			jobLabels := map[string]string{
+				"release.openshift.io/verify": "true",
```

Comment:
Is this extra label just form demonstration purposes?

Is there a spot tracking these API-like labels?

Reply:
The release-controller specifically tracks the prowjobs labeled: `"release.openshift.io/verify": "true"` 
I modified the logic to allow the release-controller to "ignore" the analysis jobs, but also add something specific for you to key off of.

I am more than happy to chage/update this to whatever you would prefer.  I left a `TODO` to make sure that we're in agreement on how to proceed here.